kroto-plus-coroutines/src/test/kotlin/com/github/marcoferrer/krotoplus/coroutines/integration/ClientStreamingBackPressureTests.kt

```diff
@@ -74,10 +75,10 @@ class ClientStreamingBackPressureTests {
 
     @Test
     fun `Client send suspends until server invokes receive`() {
-        lateinit var serverRequestChannel: ReceiveChannel<HelloRequest>
```

Comment:
This change was to prevent race conditions between the `serverRequestChannel` var being populated and test assertions running.